assets/charts/components/azure-arc-onboarding/templates/remove.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  annotations:
+    "helm.sh/hook": "pre-delete"
+  name: azure-arc-remove
+spec:
+  template:
+    spec:
+      serviceAccountName: azure-arc-onboarding
+      automountServiceAccountToken: true
+      containers:
+      - image: quay.io/kinvolk/az-cli-with-helm:v0.1
+        name: install-azure-arc
+        args:
+          - remove
+        env:
+          - name: AZURE_APPLICATION_CLIENT_ID
+            value: {{ .Values.applicationClientID }}
```

Comment:
Shall we use `required` here to ensure this value is not empty? (also for other fields)

Reply:
done.